### PR TITLE
Use released version of documenteer.

### PR DESCRIPTION
**Pull request by ktlim**

### CODE REVIEW

**Comment by timj** [comment]
@ktlim I'm fine with this since Jonathan says he won't have instructions available for migration until next week.

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -49,7 +49,7 @@ jobs:
         run: pytest -r a -v -n 3 --open-files
 
       - name: Install documenteer
-        run: pip install documenteer[pipelines]
+        run: pip install documenteer[pipelines]==0.5.8
```
Comment (timj) [inline .github/workflows/build.yaml]: I think <0.6.0 is also an option.

**Review by TallJimbo — APPROVED** [review]
I'm all for just merging this.

**Comment by ktlim** [comment]
I've got to get the syntax right first :)

**Pull request merged**